packages/application-extension/src/index.tsx

```diff
@@ -1060,6 +1063,63 @@ const topSpacer: JupyterFrontEndPlugin<void> = {
   }
 };
 
+/**
+ * The simple interface mode switch in the status bar.
+ */
+const modeSwitchPlugin: JupyterFrontEndPlugin<void> = {
+  id: '@jupyterlab/statusbar-extension:mode-switch',
```

Comment:
Looks like we could also rename the plugin id, for example:

```suggestion
  id: '@jupyterlab/application-extension:mode-switch',
```